schemas/develops.json

```diff
@@ -0,0 +1,56 @@
+{
+  "$id": "resource:/develops.json",
+  "title": "product",
+  "description": "Schema to describe a product at lobid.",
+  "type": "array",
+  "items": {
```

Comment:
I'd start with a simple not qualified `develops` relation here. Also, looking at this, I am unsure about the name, as one would initially think this is about a group that develops software and not about a project that develops further a product...

Reply:
We just discussed to use `enhances` here.